### PR TITLE
Works and About me sections

### PR DESCRIPTION
**Pull request by eamador1**
Since the last code review, the following changes were made in this project:

- [ ] Add section about my work
- [ ] Add section about me
- [ ] Check linters

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -37,6 +34,223 @@ <h4 class="profession">I am a Software Developer</h4>
       <li class="list-social"><a class="logos"  href="#"><img src="img/twitter.svg" alt="Twitter Logo"></a></li>
       <li class="list-social"><a class="logos"  href="#"><img src="img/medium.png" alt="Medium Logo"></a></li>
     </ul>         
-  </section>  
+  </section> 
+  
+  <section class="works">
+    <header class="container-header-works">
+      <h3 class="header-works">My Recent Works</h3>
+      <hr class="division division-line1"></hr>
```
Comment (souad988) [inline index.html:42]: Great job so far 👍🏼 , I want just to mention that this divider is not well aligned kindly refer to the images below and fix the  issue 
image (31)
image (32)

File: style.css [hunk 2]
```diff
@@ -149,3 +149,310 @@ html,
...
+  margin-right: 10%;
+  justify-content: center;
+  align-content: center;
+  gap: 2vh;
+  margin-top: 2vh;
+}
+
+.container-cards {
+  width: 100vw;
+  display: flex;
```
Comment (souad988) [inline style.css:213]: Good work , you cards are well aligned but I would ask you to use grid layout here as it was requested in this project requirement : "in order to lay out the elements in "Works" section you should use CSS Grid and Flexbox combined (it is mandatory that you use both). Use CSS Grid in the main container that contains all of the cards and use Flexbox to lay out an individual card"

**Review by souad988 — CHANGES_REQUESTED** [review]
Hi @eamador1 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

242722065-db428b0c-8dfc-49c4-a156-42f227e32868


## Highlights
Good PR description
Use of semantic tags

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by eamador1** [comment]
Since the last review, the following changes were made in this project:

- [ ] Add left-margin to division line
- [ ] Add grid layout to cards

**Review by christianbiring1 — APPROVED** [review]
Hi @eamador1 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
# STATUS: APPROVED :tada: 

## Highlights
- [ ] Great work on implementing the changes as requested by the previous reviewer :+1: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**